Adding @cockroachlab/ui-components and restyling Home

nathanstilwell opened the pull request:
Added a dependency on @cockroachlabs/ui-components and reworked Home.tsx
to align with designs in Figma.

![Screen Shot 2022-03-23 at 10 58 09 AM](https://user-images.githubusercontent.com/397448/159729962-5d9f4afe-dc66-4414-9feb-a4671c5940a2.png)

![Screen Shot 2022-03-23 at 10 58 21 AM](https://user-images.githubusercontent.com/397448/159729993-dbcb64c6-e787-4e1e-bb01-ab418070eeaf.png)

![Kapture 2022-03-23 at 11 00 06](https://user-images.githubusercontent.com/397448/159729825-ea2ff338-20b7-48d2-871d-b3448e63da4d.gif)

Commit:
Adding @cockroachlab/ui-components and restyling Home

Added a dependency on @cockroachlabs/ui-components and reworked Home.tsx
to align with designs in Figma.

Annebirzin reviewed (approved):
Looking great! I have 2 very minor nits that are not blockers:
- In the primary nav, can we nudge the `SQL Migration Tool` text down so it is vertically aligned with the CRDB logo? (ie. bump it down so it sits on the same baseline as `CockroachDB`) 
- Can we left-align the `SQL importer` header with the descriptions and buttons?

Commit:
Spacing adjustments to Home, container width improvements

- Added some styles to align heading elements and surrounding content
- Added a min-width (640px) and a responsive width (80vw) to Container

nathanstilwell commented:
@Annebirzin Thanks for the review. I adjusted the heading and content alignment as well as adjusting the container width to be a little more responsive (max width 1280px, width 80% of the viewport width, min width 640px).
